index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <nav class="nav-bars">
+            <nav class="logo">
+              <h2>DNG</h2>
+            </nav>
+            <nav class="menu">
```

Comment:
- [ ] Please the toolbar section should be wrapped in a header tag.
![image](https://user-images.githubusercontent.com/74744787/185756984-bd596462-b516-4859-9f9e-beb7054f40e2.png)